### PR TITLE
Fix - TEC options not working properly when RSVP email has `Use Ticket Email Settings` -> On

### PR DESCRIPTION
**Pull request by rafsuntaskin**
TE-AB23
TE-B6CA

Screencast:
**Test with Preview only**

https://github.com/the-events-calendar/the-events-calendar/assets/7523321/44a28a5d-fb49-4208-b611-ef9fd0942399

**Test with Email View**

https://github.com/the-events-calendar/the-events-calendar/assets/7523321/3feb73f6-10ca-4245-b29b-e219d69b851b

### CODE REVIEW

File: src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/RSVP.php [hunk 1]
```diff
@@ -166,54 +169,4 @@ public function include_event_links( $parent_template ) {
...
-			}
-		}
-		if (
-			! empty( $args['email'] )
-			&& $args['email']->get_id() !== $rsvp_class->get_id()
-		) {
-			return false;
-		}
-
-		if ( ! empty( $args['preview'] ) && isset( $args['add_event_links'] ) ) {
```
Comment (juanfra) [inline src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/RSVP.php:201]: We must keep these checks for the preview to work as expected.

Comment (rafsuntaskin) [reply src/Events/Integrations/Plugins/Event_Tickets/Emails/Email/RSVP.php:201]: I didn't find any issues with the previews, I will adding the screencast as artifact.

**Review by redscar — APPROVED** [review]

**Comment by rafsuntaskin** [comment]
> Lets get a Artifact around making sure we test with live preview. Change the field, without saving and click on preview.

Done

**Review by bordoni — APPROVED** [review]
Thank you!

**Pull request merged**